webpack/components/extensions/HostDetails/DetailsTabCards/RegistrationCard.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import CardTemplate from 'foremanReact/components/HostDetails/Templates/CardItem/CardTemplate';
+
+export const RegisteredBy = ({ user, activationKey }) => {
+  const activationKeyID = activationKey?.id;
+  const activationKeyName = activationKey?.name;
+  if (user) {
+    return (
+      <DescriptionListDescription>{user}</DescriptionListDescription>
+    );
+  }
+  return (
+    <>
+      <Text component={TextVariants.h4}>Activation key</Text>
+      <a href={urlBuilder(`activation_keys/${activationKeyID}`, '')}>{activationKeyName}</a>
+    </>
+  );
+};
+
+RegisteredBy.propTypes = {
+  user: PropTypes.string,
+  activationKey: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+};
+
+RegisteredBy.defaultProps = {
+  user: '',
+  activationKey: {},
+};
+
+const RegistrationCard = ({ isExpandedGlobal, hostDetails }) => {
+  const registerDate = hostDetails?.subscription_facet_attributes?.registered_at;
+  const registerEndpoint = hostDetails?.subscription_facet_attributes?.registered_through;
+  const registerUser = hostDetails?.subscription_facet_attributes?.user?.login;
+  const registerActivationKey = hostDetails?.subscription_facet_attributes?.activation_keys[0];
+  if (!registerDate) return null;
+  return (
+    <CardTemplate
+      overrideGridProps={{ rowSpan: 2 }}
```

Comment:
I just learned that this is what's causing the cards to be too tall.  The default `rowSpan` is 1, so I recommend deleting this line entirely.

Can you please fix it in the Installed Products card as well since that's merged already? :angel:

Reply:
Yeah I can